napalm/ios/ios.py

```diff
@@ -2173,6 +2173,52 @@ def get_config(self, retrieve='all'):
 
         return configs
 
+    def get_ipv6_neighbors_table(self):
+        """
+        Get IPv6 neighbors table information.
+        Return a list of dictionaries having the following set of keys:
+            * interface (string)
+            * mac (string)
+            * ip (string)
+            * age (float) in seconds
+            * state (string)
+        For example::
+            [
+                {
+                    'interface' : 'MgmtEth0/RSP0/CPU0/0',
+                    'mac'       : '5c:5e:ab:da:3c:f0',
+                    'ip'        : '2001:db8:1:1::1',
+                    'age'       : 1454496274.84,
+                    'state'     : 'REACH'
+                },
+                {
+                    'interface': 'MgmtEth0/RSP0/CPU0/0',
+                    'mac'       : '66:0e:94:96:e0:ff',
+                    'ip'        : '2001:db8:1:1::2',
+                    'age'       : 1435641582.49,
+                    'state'     : 'STALE'
+                }
+            ]
+        """
+
+        ipv6_neighbors_table = []
+        command = 'show ipv6 neighbors'
+        output = self._send_command(command)
+
+        for entry in output.split('\n')[4:]:
+            # typical format of an entry in the IOS IPv6 neighbors table:
```

Comment:
Don't split on a fixed location in the file. Do a split() or an re.split() and remove all the header information before the for loop:

Something like:
```
_, neighbor_table = re.split(r^IPv6 Address.*nterface$', output, flags=re.M)
```

Note, the information like:

```
Load for five secs: 9%/5%; one minute: 9%; five minutes: 9%
Time source is NTP, 08:55:30.881 DST Tue Nov 7 2017
```

is not guaranteed to be there (i.e. it depends on your config) and will be stripped off by this method:

```
def _send_command_postprocess(output):
```

This is causing at least part of the issue with the unit tests.